src/components/ApiErrors/index.tsx

```diff
@@ -51,20 +52,18 @@ const ApiErrors = React.forwardRef<HTMLDivElement, ApiErrorsProps>(
         "<br />"
       )[2];
     } else if (lang !== "en" && typeof problemDetail !== "string") {
-      const errorToTranslate = problemDetail?.detail;
+      const errorToTranslate = problemDetail.detail;
       let newErrorMessage;
       try {
         newErrorMessage = errorToTranslate
           ? apiErrorTranslations[errorToTranslate][lang]
           : t("apiErrors.defaultError");
       } catch (e) {
-        console.log(
+        logger.error(
```

Comment:
`logger` only works on server side, I think we can switch it back to `console.log` here to fix the CI/CD